src/bundle/Resources/public/js/CKEditor/core/base-ckeditor.js

```diff
@@ -111,6 +112,26 @@ import ContextualBalloon from '@ckeditor/ckeditor5-ui/src/panel/balloon/contextu
             const wrapper = this.getHTMLDocumentFragment(container.closest('.ez-data-source').querySelector('textarea').value);
             const section = wrapper.childNodes[0];
             const customTags = Object.keys(window.eZ.richText.customTags);
+            const blockCustomStyles = Object.entries(eZ.richText.customStyles)
+                .map(([customStyleName, customStyleConfig]) => {
+                    if (customStyleConfig.inline) {
```

Comment:
Personally I would first do filter and later map, that way you could even avoid this return here